src/Aspire.Hosting.Keycloak/KeycloakResourceBuilderExtensions.cs

```diff
@@ -146,20 +146,26 @@ public static IResourceBuilder<KeycloakResource> WithDataBindMount(this IResourc
     /// </example>
     public static IResourceBuilder<KeycloakResource> WithRealmImport(
         this IResourceBuilder<KeycloakResource> builder,
-        string importDirectory,
+        string import,
```

Comment:
This is a breaking change if you use named parameters but this package is preview so care level is low 😄

Reply:
> This is a breaking change if you use named parameters but this package is preview so care level is low 😄

Yes and yes. But I couldn't keep `importDirectory` when the semantics has changed.